lib/widgets/teach_word/word_vocab_content.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class WordVocabContent extends StatelessWidget {
+  final String vocab;
+  final String meaning;
+  final String sentence;
+  final double imgSize;
+  const WordVocabContent({
+    Key? key,
+    required this.vocab,
+    required this.meaning,
+    required this.sentence,
+    required this.imgSize,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Text(vocab,
+            textAlign: TextAlign.center,
+            style: const TextStyle(
+              fontSize: 45,
+              fontFamily: "Serif",
+              color: Color.fromRGBO(245, 245, 220, 100),
+            )),
+        Padding(
+          padding: const EdgeInsets.all(10.0),
+          child: Text.rich(
+              textAlign: TextAlign.start,
+              TextSpan(
+                  text: "解釋：\n",
+                  style: const TextStyle(
+                    height: 1.1,
+                    fontSize: 18,
+                    color: Color.fromRGBO(228, 219, 124, 1),
+                    fontFamily: "Serif",
```

Comment:
Remove all the code with `fontFamily: "Serif",`, because we have set all the words to this fontfamily in main.dart using ThemeData.